tests/buyProduct_test.js

```diff
@@ -22,43 +22,53 @@ let addressees = {
     city: "Kyiv",
 };
 
+const LinksGetter = require('../helpers/productLinksGetter.js');
+const home = require("../pages/home");
+let productLinks3 = LinksGetter.getLinks();
+console.log(productLinks3);
+
 Feature('buy product');
 
-Scenario('buy product', async ({ I, productPage, checkoutPage }) => {
+Before(async ({ I, homePage }) => {
     I.login(loginUser);
-    I.openProductPage();
+    await homePage.emptyCart();
 
-    let price = await productPage.getProductPrice();
-    console.log("Product price: " + price);
+});
 
-    productPage.selectColorProduct();
+Data(productLinks3).Scenario('buy product', async ({ I, productPage, checkoutPage, orderHistoryPage, current, }) => {
+    console.log(current.link)
 
-    let colorPrice = await productPage.getColorProductPrice();
-    console.log("Color Price: " + colorPrice);
+    I.amOnPage(current.link);
+    let price = await productPage.getProductPrice();
+    console.log("Product price: " + price),
 
-    I.cardProcess();
-    I.changeAddress();
-    checkoutPage.step2BillingDetails(buyer);
-    I.changeShippingAddress();
-    checkoutPage.step3DeliveryDetail(addressees);
-    checkoutPage.deliveryMethod();
-    checkoutPage.paymentMethod();
+        productPage.cardProcess();
 
-    let deliveryPrice = await checkoutPage.getProductDeliveryPrice();
-    console.log("Delivery price: " + deliveryPrice);
+    let itemAvailable =
```

Comment:
rename to itemUnavailable to correctly display state. Currently `true` means item is not available, which is opposite to name of variable